website/rspress.config.ts

```diff
@@ -316,6 +317,112 @@ function getSidebarConfig(lang: 'zh' | 'en'): Sidebar {
 				link: getLink('/config/other-options'),
 			},
 		],
+		[getLink('/plugins/')]: [
+			{
+				text: getText('简介', 'Introduction'),
+				link: getLink('/plugins'),
+			},
+			{
+				text: getText(
+					'同步自 webpack 的内置插件',
+					'Synchronized Built-in Plugins from webpack'
```

Comment:
Can we make the title a bit shorter..

![image](https://github.com/web-infra-dev/rspack/assets/7237365/54b5338e-26b5-4f42-9241-20c9bbe15735)

Reply:
Any suggestions?

Reply:
How about "Webpack-aligned Built-in Plugins" and "Rspack-only Built-in Plugins":

![image](https://github.com/web-infra-dev/rspack/assets/7237365/2dcc8d12-9b4b-4372-84b1-81538a5a0ff0)